src/averageAges.js

```diff
@@ -14,12 +14,30 @@
  *
  * @return {number}
  */
+
+function callBack(arr, len) {
```

Comment:
```suggestion
function calculateAverageAge(ages) {
```
Let's use more descriptive and meaningful names. 
Also, why do you need to pass the array length? You can get it inside the function